subsys/net/ip/tcp.c

```diff
@@ -3345,13 +3345,20 @@ uint16_t net_tcp_get_supported_mss(const struct tcp *conn)
 	if (family == AF_INET) {
```

Comment:
In commit "net: tcp: Fix IPv6 TCP MSS issue" could you add more information what/why is being fixed to the commit message. This way user does not necessarily go to github to see the explanation.
We can still have the pointer to the issue, so you could add
"Fixes: #56657"
to the commit message too.